scripts/core/superdesk-api.d.ts

```diff
@@ -216,7 +216,7 @@ declare module 'superdesk-api' {
 
         sideWidget: null | {
             id: string;
-            pinned: boolean;
+            pinned?: boolean;
```

Comment:
why are you making it optional?

Reply:
Because it can be `null`. Type was wrong

Reply:
I see.